temporal-testing/src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -245,9 +249,43 @@ private StartWorkflowExecutionRequest overrideStartWorkflowExecutionRequest(
     if (taskTimeoutMillis != Durations.toMillis(request.getWorkflowTaskTimeout())) {
       request.setWorkflowTaskTimeout(Durations.fromMillis(taskTimeoutMillis));
     }
+
+    request.setSearchAttributes(getValidSearchAttributes(request));
     return request.build();
   }
 
+  private SearchAttributes.Builder getValidSearchAttributes(
```

Comment:
It's not a getter, please rename. It can be supply, convert, extract, transform, but not get.